File: ReciTree/Models/Recipe.cs
namespace ReciTree.Models
{
    public class Recipe : RepoItem<int>
    {
        public string Name { get; set; }
        public string Instructions { get; set; }
        public string InstructionsPic { get; set; }
        public string CreatorId { get; set; }
        public Boolean IsPrivate { get; set; } = false;
        public Profile Creator { get; set; }
    }
}

File: ReciTree.client/src/models/Recipe.js
import { Profile } from "./Account"

export class Recipe {
    constructor(data) {
        this.id = data.id
        this.name = data.name
        this.instructions = data.instructions
        this.instructionsPic = data.instructionsPic
        this.creatorId = data.creatorId
        this.img = data.img
        this.creator = new Profile(data.creator)
        this.category = data.category
        this.isPrivate = data.isPrivate
    }
}